meson.build

```diff
@@ -15,37 +15,39 @@ cppcompiler = meson.get_compiler('cpp').get_argument_syntax()
 if not meson.is_subproject()
 
 # テストプロジェクト用の設定追加
+ldflags = []
 if cppcompiler == 'msvc'
     # VSプロジェクトに編集しうるファイルを追加する
     vs_files = ['include/rivet.hpp']
-    options = ['/std:c++latest', '/source-charset:utf-8', '/Zc:__cplusplus']
+    cxxflags = ['/std:c++latest', '/source-charset:utf-8', '/Zc:__cplusplus']
 elif cppcompiler == 'gcc'
     vs_files = []
-    options = []
+    cxxflags = []
     if meson.get_compiler('cpp').get_id() == 'clang'
-        options = ['-stdlib=libc++', '-fexperimental-library']
+        cxxflags = ['-stdlib=libc++', '-fexperimental-library']
+        ldflags = ['-stdlib=libc++', '-fexperimental-library']
```

Comment:
`options` was ambiguous so I changed it according to the GNU Makefile naming conventions `CXXFLAGS` and `LDFLAGS`.

Reply:
I think it is good. I will do the same from now on!